.drone.yml

```diff
@@ -144,4 +144,4 @@ pipeline:
             - ./deploy.sh
         when:
             event: deployment
-            environment: pr
+            environment: [pr]
```

Comment:
Is there a particular reason for the square brackets here as there is no sequence of values?

Reply:
I had added another namespace to that step, so they have to be in a list. But then I removed it, so it no longer needs to be